content/en/docs/concepts/storage/volumes.md

```diff
@@ -14,21 +14,32 @@ weight: 10
 
 <!-- overview -->
 
-On-disk files in a container are ephemeral, which presents some problems for
+Kubernetes volumes provide a way for containers in a Pod to retain
+data beyond a container's lifecycle. They act as directories that can store data
+and are accessible across containers in the Pod.
```

Comment:
This isn't right. The things that let you retain data are PersistentVolumes.

Reply:
```suggestion
Kubernetes _volumes_ provide a way for containers in a Pod to access and share data via the
filesystem. There are different kinds of volume that you can use for different purposes, such
as:

- defining a configuration file based on a ConfigMap or a Secret
- providing some temporary scratch space for a pod
- sharing a filesystem between two different containers in the same pod
- sharing a filesystem between two different pods (even if those Pods run on different nodes)
- durably storing data so that it stays available even if the Pod restarts or is replaced
- passing configuration information to an app running in a container, based on details of the Pod
  the container is in
  (for example: telling a {{< glossary_tooltip text="sidecar container" term_id="sidecar-container" >}}
  what namespace the Pod is running in)
- providing read-only access to data in a different container image

Data sharing can be between different local processes within a container, or between different containers,
or between Pods.
```

Reply:
Ah, I see. I took a second look and you are correct.

I like this suggestion, it provides several reasons why a volume is used.